mkdocs.yml

```diff
@@ -6,8 +6,6 @@ site_author: developer@fetch.ai,info@valory.xyz
 
 theme:
   name: 'material'
-  logo:
-    icon: code
```

Comment:
This is currently causing the issues with the bumped version of mkdocs.

I have checked the docs;

https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#logo

Looks like the syntax is now wrong.

I couldnt find the icon, otherwise I would have updated it. We are currently rendering this;

![image](https://user-images.githubusercontent.com/35799987/160699817-4e301705-8b58-4b38-8034-4ffdef0a0d81.png)

Versus prod which is;

![image](https://user-images.githubusercontent.com/35799987/160699917-70f7245e-d203-447c-9630-77cb0e326f16.png)

Reply:
ok that's no problem atm